### ISSUE TITLE
No response received from singularity hub during pull

### ISSUE THREAD

**Issue by tikk3r** [singularityhub/singularityhub.github.io#205]
### Links

 - container collection: tikk3r/lofar-grid-hpccloud
 - GitHub repository or recipe file: https://github.com/tikk3r/lofar-grid-hpccloud

### Version of Singularity

Laptop: singularity version 3.5.2-1.1.fc31
Computer 1: singularity version 3.2.1-1.1.el7
Computer 2: singularity version 3.4.2-1.1.el7

### Behavior when Building Locally

The container builds fine, both locally and on shub.

### Error on Singularity Hub

No error on singularity hub. When I run

```
singularity pull shub://tikk3r/lofar-grid-hpccloud:lofar_sksp@2002c0a6d1445d8d9b519edd3a05f25e2009e974
```
It does not work on the above mentioned laptop and computer 1. I see the following output:
```
INFO:    Downloading shub image
FATAL:   While pulling shub image: no response received from singularity hub
```

On computer 2 (version 3.4) it works fine and it starts downloading the image.

### What do you think is going on?

I have no clue, it works on one computer and not on two others. The same problem occurs for another collection and has the same behaviour working on computer 2, but not on the others. Could it be related to local setup or something like that?

**Comment by vsoch** [comment]
Hey @tikk3r ! There are a few possible things that might be going on:

 - 1: you exhausted your weekly get counts. This is more likely to happen on Sunday before they are reset. I just checked the server and they look okay right now.
 - 2: there is some change in the Singularity client that prevents the pull (you noted different versions?)
 - 3. the server is down

The server is definitely not down so we can rule out 3, and we can also rule out number 1 because I just checked and you are well below. I've also just pulled the container to my system and it works okay. I would check with Sylabs (open an issue over there) about any possible changes to the Singularity client. For example, I know that the scs-library-endpoint just had a multipart upload endpoint added, and it could be that they wound this into their shub pull without noticing.  It looks like there was some work on this file [three days ago](https://github.com/sylabs/singularity/blob/df3893ef37d5404c7495bc58bf31e5b70afce7c9/internal/pkg/client/shub/api.go#L83) something about the cache. Another idea is that it could be something about a proxy? Anyway, I would suggest opening an issue with Sylabs as this seems Singularity or host specific.
